src/fiskaltrust.Middleware.Interface.Http/HttpPos.cs

```diff
@@ -0,0 +1,214 @@
+﻿using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Interface.Http.Helpers;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace fiskaltrust.Middleware.Interface.Http
+{
+    public class HttpPos : IPOS
+    {
+        private readonly HttpPosOptions _options;
+        private readonly HttpClient _httpClient;
+
+        private delegate string AsyncEchoCaller(string message);
+        private delegate ifPOS.v0.ReceiptResponse AsyncSignCaller(ifPOS.v0.ReceiptRequest request);
+        private delegate Stream AsyncJournalCaller(long ftJournalType, long from, long to);
+
+        public HttpPos(HttpPosOptions options)
```

Comment:
Can we use this class for communciation with signaturcloud too?

Reply:
Sure :)